docker/integration-test-rust-binaries-runner/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM debian:bookworm-slim
```

Comment:
We probably don't strictly need it. It's just a bit more isolation (of mostly filesystem and env) that at least in my opinion comes relatively cheap both in terms of code complexity and execution time overhead.

Reply:
Hm. But regarding the filesystem, for example -- do I get it right that since you don't mount a separate volume for this isolated container, stuff like RocksDB is created within the image and won't be persisted? In other words, it won't be possible to restart such a node and continue working (at least without having to rebuild the caches)?

Reply:
And I'm also slightly concerned that we have to build the docker image (with stuff like `apt-get install` which is not only slow itself, but also can fail due to some network issue).

Reply:
I've changed that instead of building image from scratch I use zk-environment image.

Reply:
About the filesystem, ability to restart the node wasn't the highest priority for me, it can be added later if needed.